mendix/manifest.json

```
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "4119b134-c828-4e14-95b5-585bb13d314a",
+  "app_id": "mendix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Market leading low-code IDE for application development",
```

```suggestion
    "description": "Monitor Mendix environment metrics",
```
This should be a description of the integration rather than Mendix.

---

Please remove marketing forward language